### PR TITLE
Add docs for differential drivetrain kinematics

### PR DESCRIPTION
**Pull request by DylanB5402**
Issue #169

### CODE REVIEW

**Comment by AustinShalit** [comment]
This needs a rebase for the new layout.

**Comment by yellowjaguar5** [comment]
@DylanB5402 Are you still working on this or interested in working on this PR? If not, I can continue working on it.

File: source/docs/software/kinematics/kinematics.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Kinematics
+==========
+
+The kinematics suite simplifies advanced control of differential, mecanum, and swerve drivetrains, handling the inverse kinematics and odometry for a user. Each type of drivetrain has a kinematics and odometry class. The kinematics classes convert chasis speeds to robot wheel speeds and vice versa. The odometry classes calculate and return the position of the robot on the cartesian plane.
+
+Chasis Speeds
```
Comment (Modelmat) [inline source/docs/software/kinematics/kinematics.rst]: Chassis

**Pull request closed without merging**